script/base/DeployMorphoTokenBase.sol

```diff
@@ -0,0 +1,46 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity ^0.8.0;
+
+import "../../lib/forge-std/src/Script.sol";
+import "../../lib/forge-std/src/console.sol";
+
+import {MorphoTokenOptimism} from "../../src/MorphoTokenOptimism.sol";
+import {ERC1967Proxy} from
+    "../../lib/openzeppelin-contracts-upgradeable/lib/openzeppelin-contracts/contracts/proxy/ERC1967/ERC1967Proxy.sol";
+
+contract DeployMorphoTokenBase is Script {
+    address public constant MORPHO_DAO = 0xcBa28b38103307Ec8dA98377ffF9816C164f9AFa;
+    address public REMOTE_TOKEN;
+    address public constant BRIDGE = 0x4200000000000000000000000000000000000010;
+
+    bytes32 public IMPLEMENTATION_SALT;
+    bytes32 public PROXY_SALT;
+
+    address public implementationAddress;
+    MorphoTokenOptimism public token;
+    address public wrapperAddress;
+    address public newMorphoAddress;
```

Comment:
I know it doesnt' really matter but why are these in "storage"

Reply:
should we create a specific token address like the previous token?

Reply:
related discussion: https://github.com/morpho-org/morpho-token-upgradeable/pull/81#discussion_r1826548165

Reply:
True that they could be memory